### DISCUSSION TITLE
Import from Flickr

### DISCUSSION THREAD

**Discussion by jdaviescoates** [photoprism/photoprism#806] [Q&A] [answered]
I'd really like to switch from Flickr to photoprism...

Is it easy to import from Flickr?
Will photoprism automatically pickup all my Flickr tags?

Thanks!

**Comment by graciousgrey** [comment, marked as answer]
We do not yet support Flickr's json format - I just created  ticket for it: https://github.com/photoprism/photoprism/issues/812

What we do already is setting titles and keywords from filenames. 
E.g. a file named vacation_africa_2021_50799344566_o.jpg would get the keywords vacation and africa.

**Comment by brownphotographic** [comment]
https://exiftool.org/forum/index.php?topic=11072.0
